scripts/tests/multisig.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+set -ex
+
+cargo build
+scripts/test_env.py setup
+cd test
+
+gen_key_pair() {
+    ./kagami crypto -cs $1
+}
+
+DOMAIN="wonderland"
+
+gen_account_id() {
+    public_key=$(gen_key_pair $1 | head -n 1)
+    echo "$public_key@$DOMAIN"
+}
+
+gen_signatories() {
+    for i in $(seq 1 $1); do
+        key_pair=($(gen_key_pair $i))
+        public_key=${key_pair[0]}
+        private_key=${key_pair[1]}
+        # yield an account ID
+        echo "$public_key@$DOMAIN"
+        # generate a config
+        cat client.toml | sed '/domain/d' | sed '/public_key/d' | sed '/private_key/d' > client.$i.toml
+        echo "domain = \"$DOMAIN\"" >> client.$i.toml
+        echo "public_key = \"$public_key\"" >> client.$i.toml
+        echo "private_key = \"$private_key\"" >> client.$i.toml
+    done
+}
+
+# populate signatories
+N_SIGNATORIES=3
+SIGNATORIES=($(gen_signatories $N_SIGNATORIES))
+for signatory in ${SIGNATORIES[@]}; do
+    ./iroha account register --id $signatory
+done
+
+# register a multisig account
+MULTISIG_ACCOUNT=$(gen_account_id "msa")
+WEIGHTS=($(yes 1 | head -n $N_SIGNATORIES)) # equal votes
+QUORUM=$N_SIGNATORIES # unanimous
+TRANSACTION_TTL="1y 6M 2w 3d 12h 30m 30s 500ms"
```

Comment:
IMO this is pretty unusable to CLI users. I haven't seen CLI tools with such API.
I'd much rather you revert to using seconds

Reply:
There seems to be a disagreement between reviewers. I don't have a strong opinion on this. May I leave this to #4930 @mversic  @0x009922  ?

Reply:
Yeah, I totally disagree with Marin on that. @nxsaken @DCNick3 what do you think?

Anyway, I am okay with postponing it and using any solution for now.